pkg/trace/api/responses.go

```diff
@@ -27,7 +28,8 @@ const (
 // should we add another fied.
 type traceResponse struct {
 	// All the sampling rates recommended, by service
-	Rates map[string]float64 `json:"rate_by_service"`
+	Rates      map[string]float64 `json:"rate_by_service"`
```

Comment:
Thinking of adding `omitempty` here. But, what if older tracers always expect `rate_by_service` to exist in the response, and crash when it doesn't exist?

Reply:
yes keep it as before in case

Reply:
I think it will be clearer if you return an explicit empty json map. I suppose it's better to keep a json in the response in case we add other fields in the future